ElementX/Sources/Screens/CreateRoom/View/CreateRoomScreen.swift

```diff
@@ -0,0 +1,187 @@
+        }
+        .formSectionStyle()
+    }
+    
+    private var topicSection: some View {
+        Section {
+            TextField(L10n.screenCreateRoomTopicLabel,
+                      text: $context.roomTopic,
+                      prompt: Text(L10n.screenCreateRoomTopicPlaceholder),
+                      axis: .vertical)
+                .accessibilityIdentifier(A11yIdentifiers.createRoomScreen.roomTopic)
+                .lineLimit(3, reservesSpace: false)
+        } header: {
+            Text(L10n.screenCreateRoomTopicLabel)
+        } footer: {
+            if !context.viewState.selectedUsers.isEmpty {
+                selectedUsersSection
+            }
+        }
+        .formSectionStyle()
+    }
+    
+    @State private var frame: CGRect = .zero
+    @ScaledMetric private var invitedUserCellWidth: CGFloat = 64
+
+    private var selectedUsersSection: some View {
+        ScrollView(.horizontal, showsIndicators: false) {
+            LazyHStack(spacing: 28) {
+                ForEach(context.viewState.selectedUsers, id: \.userID) { user in
+                    InviteUsersScreenSelectedItem(user: user, imageProvider: context.imageProvider) {
+                        context.send(viewAction: .deselectUser(user))
+                    }
+                    .frame(width: invitedUserCellWidth)
+                }
+            }
+            .padding(.init(top: 16, leading: 32, bottom: 16, trailing: 32))
+        }
+        .frame(width: frame.width)
+    }
+    
+    private var securitySection: some View {
+        Section {
+            Picker(L10n.commonSecurity, selection: $context.isRoomPrivate) {
+                Label {
+                    VStack(alignment: .leading, spacing: 8) {
+                        Text(L10n.screenCreateRoomPrivateOptionTitle)
+                        Text(L10n.screenCreateRoomPrivateOptionDescription)
+                            .font(.compound.bodyMD)
+                            .foregroundColor(.element.secondaryContent)
+                    }
+                } icon: {
+                    Image(systemName: "lock.shield")
+                }
+                .tag(true)
+                .labelStyle(FormRowLabelStyle(alignment: .top))
+                Label {
+                    VStack(alignment: .leading, spacing: 8) {
+                        Text(L10n.screenCreateRoomPublicOptionTitle)
+                        Text(L10n.screenCreateRoomPublicOptionDescription)
+                            .font(.compound.bodyMD)
+                            .foregroundColor(.element.secondaryContent)
+                    }
+                } icon: {
+                    Image(systemName: "exclamationmark.shield")
+                }
+                .tag(false)
+                .labelStyle(FormRowLabelStyle(alignment: .top))
+            }
+            .labelsHidden()
+            .pickerStyle(.inline)
+        } header: {
+            Text(L10n.commonSecurity.uppercased())
```

Comment:
I don't think the `.uppercased()` is needed here, the default style does that automatically right?
```suggestion
            Text(L10n.commonSecurity.uppercased())
                .formSectionHeader()
```